distributionviewer/core/static/css/lib.styl

```diff
@@ -17,6 +17,27 @@ $lrg-padding = 2 * $med-padding;
 $chart-preview-height = 250px;
 $chart-preview-width = 300px;
 
+/*
+Population line colors
+
+Colors selected by playing around with [1] with $mild-blue as the base color. To
+be honest, I don't know much about color theory, but these looked like they fit
+pretty well with our colorscheme.
+
+[1] http://www.colorschemer.com/online.html
+
+- openjck
+*/
+$max-populations = 8;
```

Comment:
I tried implementing this after all, but had some issues with it. I'll take another look when we add another population.